blocks/index.ts

```diff
@@ -1,19 +1,27 @@
 import { getblock, getblockcount, getblockhash, getrawtransaction } from "../jsonrpc/index.ts";
 
+import { IBlock } from "../common/interfaces.ts";
+import { hexToAscii } from "../common/utils.ts";
 import config from "../config/config.ts";
 import miners from "./miners.ts";
-import { hexToAscii } from "../utils.ts";
-
-export interface IBlock {
-  height: number;
-  signals: boolean | undefined;
-  miner: string | undefined;
-  minerWebsite: string | undefined;
-}
 
 let blocks: IBlock[] = [];
 
-async function createBlock(height: number): Promise<IBlock> {
+async function createFakeBlock(height: number): Promise<IBlock> {
+  const minerKeys = Object.keys(miners.coinbase_tags);
+  const randomIndex = Math.floor(Math.random() * minerKeys.length);
+  const randomKey = minerKeys[randomIndex];
+  const minerData = miners.coinbase_tags[randomKey];
+
+  return await Promise.resolve({
```

Comment:
nit: You don't need to return a resolved promise if your function is `async`.

Reply:
I only did this because the linter was complaining very much and making the entire function unreadable because I had a function marked as `async` but didn't use `await` inside it.

Reply:
![2021-05-01-201855_556x254_scrot](https://user-images.githubusercontent.com/1653275/116797284-81029080-aaba-11eb-9d0b-eae656fc372e.png)